api/routes/router.go

```diff
@@ -26,6 +26,7 @@ func SetupRoutes(engine *gin.Engine) {
 
 	// Loomies
 	engine.POST("/loomies/near", middlewares.MustProvideAccessToken(), controllers.HandleNearLoomies)
+	engine.GET("/loomies/exists", middlewares.MustProvideAccessToken(), controllers.HandleValidateLoomieExists)
```

Comment:
Usually `GET` routes doesn't receive any payload / JSON content, so, this endpoint should be `POST`.